examples/utils/bundle-size-tests/src/test/checkSizes.spec.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+import { readFileSync } from "node:fs";
+
+// Since bundle size reporter is broken, and doesn't block merges, do a sanity check here
+describe("checkSizes", () => {
+	it("sharedTreeAttributes", () => {
+		// This test must be run after webpack.
+		const bundle = readFileSync("./dist/sharedTreeAttributes.js", "utf-8");
+
+		// Make sure it contains something
+		assert(bundle.length > 10);
```

Comment:
Is this the standard way we validate this? Should we document the values we're using?

Reply:
This line has a comment saying what this check is doing. What more are you looking for here documentation wise?

What are you asking if this is standard about? Is the standard way to check if a bundle actually has some stuff in it to check its length against a minimum?

Reply:
I guess I'm specifically asking about the magic numbers - where are 10 and 1000 coming from? How did we pick them? Do we use them anywhere else for similar checks?

Reply:
they were selected such that if we regress and put the whole tree implementation back in this bundle, or even a large part of it, the test will fail, and also fail if we break the bundle setup so it actually pulls in nothing making the test invalid. They are really just magic numbers which only occur here and are based on the current bundle size and the likely mistakes that could change it wo would like to catch (mainly the bundle breaking such that it contains nothing, as documented on the minimum size) and breajking to contain too much (as documented on the max size)